src/verilog/verilog_parser.h

```diff
@@ -118,7 +117,9 @@ class verilog_parsert:public parsert
     return scope_ptr == nullptr ? false : scope_ptr->is_type;
   }
 
-  unsigned dummy_id;
+  // These are used for anonymous gate instances
+  // and to create a unique identifier for enum types.
+  std::size_t dummy_id = 0;
 
   std::string get_dummy_id()
```

Comment:
How about also changing the name of this method, in keeping with my comment about the `dummy_id` variable?